src/pages/Signup/Signup.jsx

```diff
@@ -0,0 +1,109 @@
+import "./Signup.css";
+import { BsArrowRight } from "react-icons/bs";
+import { postNewUser } from "../../services";
+import { useUser } from "../../context";
+import { Link, useNavigate } from "react-router-dom";
+
+export const Signup = () => {
+  const { setUser } = useUser();
+  const navigator = useNavigate();
+
+  const signupHandler = async (e) => {
```

Comment:
good to see  you used iffi function

Reply:
Thanks, don't know why but it's not working on deployed link but working on localhost. Will resolve it soon.